src/platform_impl/linux/taskbar/xapps.rs

```diff
@@ -0,0 +1,162 @@
+use std::ffi::CStr;
```

Comment:
is this implementation tested? which distro can I test this on?

Reply:
Not tested but tested to have no panics due to this. It is basically the leftover code when I thought that is mainly xapps onstead pf unity protocol. But should work if some distro uses the xapps protocol (I need to research on that as I am a windows 10 & 11 pc user only)

Reply:
Update
- Implementation will be dropped in next commit to the pr